src/System.Management.Automation/engine/NativeCommandProcessor.cs

```diff
@@ -909,25 +909,22 @@ internal override void Complete()
                         ActionPreference.SilentlyContinue,
                         out _);
 
-                    if (errorActionPref != ActionPreference.Ignore)
```

Comment:
One consequence of rely on the `Ignore` check in `_WriteErrorSkipAllowCheck()` is that the error could still be logged (`this.PipelineProcessor.LogExecutionError` & `MshLog.LogCommandHealthEvent`) even though the EAP is set to Ignore.  I suppose that is OK?

Reply:
Yeah I noticed that too...

Reply:
If that's how error records work today when they are ignored by EAP `Ignore`, then I think it's fine to follow that behavior.